README.md

```diff
@@ -88,6 +88,8 @@ You will need to set up headers with both:
 
 _If you're new to GraphQL, you can checkout [Artsy's GraphQL Workshop](https://github.com/artsy/graphql-workshop)._
 
+For `GraphQL Endpoint`, set it to `http://localhost:3000/` if you want to query for something that lives in v1 of the API, otherwise, set it to `http://localhost:3000/v2`.
```

Comment:
```suggestion
For `GraphQL Endpoint`, set it to `http://localhost:3000/` if you want to query for something that lives in v1 of the schema, otherwise, set it to `http://localhost:3000/v2` 

**Note that `/v2` is the default** and it is rare to query against `/v1`. 
```

Reply:
Wanted to note that it is `v1` of the schema. By using "API" that might confuse devs about, say, gravity v1, etc.

Reply:
`schema`. got it. i have yet to get my mp/graphql-speak down pat. thanks!